ooiod/blobs.py

```diff
@@ -15,9 +15,9 @@ def _list_blob_set(storage_account, container, nextmarker = None):
 
 def list_blobs(storage_account, container):
     blobs = []
-    blob_set, nextmarker = _list_blob_set('ooiopendata', 'camhd')
+    blob_set, nextmarker = _list_blob_set('ooiopendata', container)
```

Comment:
I think it would be nice to also generalize ooiopendata to storage_account here and below.

Reply:
Okay. I see how did not need to include the code... and should have used the code blocks if I had. I will learn how to exclude files from a pull request for my next PR.

Reply:
Roger that. This comment here was aimed at one line of code (the one highlighted). You can reply to the general comments in the main thread below.